src/index.html

```diff
@@ -7,12 +7,64 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__image"
+        src="./images/imac.jpeg"
+        alt="card image"
+      />
+
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="card__code">Product code: 195434</p>
+
+      <div class="card__content align">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="review">Reviews: 5</p>
+      </div>
+
+      <div class="card__content price">
+        <p class="price__label">Price:</p>
+        <p class="price__value">$2,199</p>
+      </div>
```

Comment:
```suggestion
      <div class="card__price">
        <p class="card__price-label">Price:</p>
        <p class="card__price-value">$2,199</p>
      </div>
```
Now you write price as a separate BEM block, same for `review` and `align`. They all should be elements of card

![image](https://github.com/user-attachments/assets/196663e4-328b-4b9f-b316-981471eeb650)

P.S. `align` is bad element name - it should describe what exactly it is